docs/querying/operators.md

```diff
@@ -205,12 +205,17 @@ or
 
     <aggr-op>([parameter,] <vector expression>) [without|by (<label list>)]
 
-`parameter` is only required for `count_values`, `quantile`, `topk` and
-`bottomk`. `without` removes the listed labels from the result vector, while
+`label list` is a list of unquoted labels that may include a trailing comma, i.e.
+both `(label1, label2)` and `(label1, label2,)` are valid syntax.
+
+`without` removes the listed labels from the result vector, while
 all other labels are preserved the output. `by` does the opposite and drops
 labels that are not listed in the `by` clause, even if their label values are
 identical between all elements of the vector.
 
+`parameter` is only required for `count_values`, `quantile`, `topk` and
```

Comment:
This makes it seem as though it's an option for the others. I'd say used rather than required.

Reply:
I personally find "required" more clear than "used" here. 

Also this wording was not introduced by this PR.